### PR TITLE
Feature/topics page

### PR DESCRIPTION
**Pull request by Gutts-n**
Adapted paginator, loading components colors, and font colors and wheight to follow guidelines.

### CODE REVIEW

File: components/_shared/carousel/icon_card/IconCard.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const IconCard: React.FC<IconCardProps> = ({
     <>
       <div
         className={`${
-          isActive ? 'text-[#22B373]' : ''
+          isActive ? 'text-[gold]' : ''
```
Comment (demenech) [inline components/_shared/carousel/icon_card/IconCard.tsx:29]: image

IMO this shade of yellow looks bad against white backgrounds. Maybe we can make this a bit darker? CC: @lucasmbispo

Comment (demenech) [reply components/_shared/carousel/icon_card/IconCard.tsx:29]: on the other hand you could also increase the font weight

**Review by demenech — APPROVED** [review]
LGTM (but @lucasmbispo is the official reviewer)

**Review by lucasmbispo — CHANGES_REQUESTED** [review]
LGTM, except for org image in mobile that shouldn't shrink.
Let's aim for 320px mobile
image

**Review by demenech — APPROVED** [review]
LGTM

**Pull request merged**